resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json

```diff
@@ -1,115 +1,407 @@
+      "Sid": "GetAndCreateTirosQueries",
+      "Effect": "Allow",
+      "Action": [
+        "tiros:CreateQuery",
+        "tiros:GetQueryAnswer",
+        "tiros:GetQueryExplanation"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "CreateRedHatInstanceTag",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateTags"
+      ],
+      "Resource": "arn:aws:ec2:*:*:instance/*",
+      "Condition": {
+        "StringEquals": {
+          "ec2:CreateAction": "RunInstances",
+          "aws:RequestTag/red-hat-managed": "true"
+        },
+        "ForAllValues:StringEquals": {
+          "aws:TagKeys": "red-hat-managed"
+        }
+      }
+    },
+    {
+      "Sid": "NetworkVerifierInstance",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:RunInstances",
+        "ec2:DescribeInstanceTypes",
+        "ec2:GetConsoleOutput"
+      ],
+      "Resource": "arn:aws:ec2:*:*:instance/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "DescribeIp",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeCoipPools",
+        "ec2:DescribeIpv6Pools",
+        "ec2:DescribePublicIpv4Pools",
+        "ec2:GetAssociatedIpv6PoolCidrs"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DescribeInternetGateway",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeEgressOnlyInternetGateways",
+        "ec2:DescribeInternetGateways"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DescribePrefixLists",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribePrefixLists",
+        "ec2:DescribeManagedPrefixLists"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "GetManagedPrefixListsEntries",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:GetManagedPrefixListEntries"
+      ],
+      "Resource": "arn:aws:ec2:*:*:prefix-list/*"
+    },
+    {
+      "Sid": "DescribeImages",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeImages"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DescribeImageAttibute",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeImageAttribute"
+      ],
+      "Resource": "arn:aws:ec2:*:*:image/*"
+    },
+    {
+      "Sid": "DescribeAddressAttribute",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeAddressesAttribute"
+      ],
+      "Resource": "arn:aws:ec2:*:*:elastic-ip/*"
+    },
+    {
+      "Sid": "DescribeAddresses",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeAddresses"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DescribeByoipCidrs",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeByoipCidrs"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DescribeKeyPairs",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeKeyPairs"
+      ],
+      "Resource": "*"
```

Comment:
These should only be key pairs for the verifeir right?

Reply:
I think the keypair being imported to the verifier to ssh in the node is the SRE own keypair. so this may be removed.